admin/package-files/docker-plugin/upstart/flocker-docker-plugin.conf

```diff
@@ -0,0 +1,11 @@
+# flocker-docker-plugin - flocker-docker-plugin job file
+
+description "Flocker Docker Plugin"
+author "ClusterHQ <support@clusterhq.com>"
+
+start on starting docker
```

Comment:
Starting automatically is not part of what we do currently for other services either I think? On CentOS I guess, but that is the default on Ubuntu so I probably forgot something.